libs/community/langchain_community/utilities/sql_database.py

```diff
@@ -373,14 +382,16 @@ def _get_sample_rows(self, table: Table) -> str:
 
     def _execute(
         self,
-        command: str,
-        fetch: Literal["all", "one"] = "all",
-    ) -> Sequence[Dict[str, Any]]:
+        command: Union[str, Executable],
+        fetch: Literal["all", "one", "result"] = "all",
```

Comment:
Is there a spot in the documentation which would need to be updated along the lines, to educate users about the new `result` fetch mode?

Reply:
We have not been able to discover a dedicated documentation section for the `SQLDatabase` adapter utility.

> Up until now, the `SQLDatabase` utility only has been demonstrated on
> behalf of the "toolkit integration" section, where it is about
> showcasing an agent designed to interact with an SQL database. [1]
>
> Contrary to that, cf57a092483b now adds a documentation page exclusively
> dedicated to the `SQLDatabase` adapter utility, and its features.
> 
> [1] https://python.langchain.com/docs/integrations/toolkits/sql_database

We hope you like it. Please provide guidance if the content is inappropriate, needs to be changed, or slotted into a different directory. Thanks.